### ISSUE TITLE
difference between google chrome and brave browser for speaker selection for google hangouts

### ISSUE THREAD

**Issue by mrose17** [brave/browser-laptop#1666]
when i'm in google hangouts, chrome and the brave report different options for speakers.

on my macbook pro, the brave browser shows only "Default speakers", but google chrome shows these additional options "Built-in Output", "DisplayPort", and "BCC950 ConferenceCam".

why the difference?

![chrome](https://cloud.githubusercontent.com/assets/400552/15167452/e80a60b2-16dd-11e6-959e-cdbfc8ee1fbf.png)
![brave](https://cloud.githubusercontent.com/assets/400552/15167458/ee08742c-16dd-11e6-94d9-019974b2388c.png)

**Comment by diracdeltas** [comment]
Does firefox show all the options too?

**Comment by mrose17** [comment]
tl;dr: yes

acts like google chrome: safari, firefox, opera developer, 

acts like brave browser:  ivaldi

thanks!

**Comment by alexwykoff** [comment]
@mrose17 is this still an issue?  Thanks!

**Comment by mrose17** [comment]
yes, i'm afraid so... try it out on your next hangout!

**Comment by mrose17** [comment]
OBE
